csfieldguide/chapters/content/en/human-computer-interaction/sections/usability-heuristics.md

```diff
@@ -285,7 +285,9 @@ Some controls try to offer the best of both worlds by having a small flap that c
 
 # Scary interfaces
 
-The following site identified some of the "scariest" interfaces around, some of which are great examples of *not* having minimalist design:  [OK/Cancel scariest interface](https://web.archive.org/web/20191216061736/http://okcancel.com:80/archives/article/2005/11/the-scariest-interface-part-ii.html).
+In 2005 Tom Chi put out a call for people to nominate the "scariest" interfaces around (the original site is [here](http://okcancel.com/archives/article/2005/11/the-scariest-interface-part-ii.html), although it wasn't working at the time of writing).
```

Comment:
Reading this and what Tim originally suggested, it seems to imply that `here` should link to the broken site. Either way it's still not correct to link an archive site as "the original site".

Perhaps just say:
> an archive of the original site is [here].

or with more explanation:
> the original site has since been taken down, but an archive of it is available [here]

Reply:
it isn't the archived site

Reply:
:facepalm: I saw 'archive' and got confused

But definitely _was_ "working at the time of writing" so maybe change that

Reply:
But it isn't working at the time of writing? :stuck_out_tongue:

Reply:
Usually the time of writing refers to the time of the article being written, not a tiny amendment to it to say that it's no longer working

Reply:
I like the current sentence though as it tells the user that the link wasn't working when we wrote this sentence, but it may work now.

I'm happy to change it I just don't think it should be "Working at the time of writing".

Reply:
nono don't say it is working at the time of writing, just also don't say it wasn't - avoid the time of writing all together. "although it may be unavailable" for example

Reply:
ready for re-review